extension/commercial_data/README.md

```diff
@@ -0,0 +1,416 @@
+
+#### Example
+
+Example response from `GET collections/PLEIADES/queryables`:
+```json
+{
+   "$schema":"http://json-schema.org/draft-07/schema#",
+   "$id":"http://example.com/commercial_data/PLEIADES/queryables",
+   "type":"object",
+   "title":"Queryables of Airbus Pleiades collection.",
+   "description":"Available properties for CQL filtering of products.",
+   "properties":{
+      "processing_level":{
+         "title":"Processing level",
+         "description":"When searching, you will receive results from the full catalog as well as the Living Library, which holds images that have cloud cover under 30% and Incidence angle under 40°. If you want to search only Living Library results, you will need to filter using processingLevel. This value could be equal to SENSOR (images which meet Living Library criteria) and ALBUM (images that do not meeting Living Library criteria in terms of incidence angle and cloud cover).'",
+         "type":"string",
+         "enum":[
+            "SENSOR",
+            "ALBUM"
+         ]
+      },
+      "max_snow_coverage":{
+         "title":"Maximum snow coverage",
+         "description":"The maximum allowable snow coverage in percent.",
+         "type":"number",
+         "minimum":0,
+         "maximum":100,
+         "default":100
+      },
+      "max_incidence_angle":{
+         "title":"Maximum incidence angle",
+         "description":"The maximum allowable incidence angle in degrees.",
+         "minimum":0,
+         "maximum":90,
+         "default":90
+      }
+   }
+}
+```
+
+### Searching available products
+
+Backends should implement top-level `GET /search` endpoint as specified in [STAC Item Search API specification](https://github.com/radiantearth/stac-api-spec/tree/v1.0.0-rc.1/item-search). This should include the [Filter Extension](https://github.com/radiantearth/stac-api-spec/tree/v1.0.0-rc.1/fragments/filter), which enables filtering the available products by attributes specified in `/queryables` and `collections/{collection-id}/queryables`. 
+The endpoint should return a list of  `STAC Item`s that match the criteria.
+
+#### Example
+
+Example request payload to `GET /search` for `PLEIADES` products from "Living Library" with no snow coverage.
+
+```json
+{
+  "bbox": [
+    3,
+    15,
+    4,
+    16
+  ],
+  "datetime": [
+    "01-01-2022",
+    "01-02-2022"
+  ],
+  "collectionsArray": [
+    "PLEIADES"
+  ],
+  "filter": {
+    "op": "and",
+    "args": [
+      {
+        "op": "=",
+        "args": [
+          {
+            "property": "processing_level"
+          },
+          "SENSOR"
+        ]
+      },
+      {
+        "op": "=",
+        "args": [
+          {
+            "property": "max_snow_coverage"
+          },
+          0
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Ordering products
+
+Backends should implement the following endpoints:
+
+- `GET /orders`: Get a list of all created orders
```

Comment:
I'm still wondering whether all endpoints should generally live under `/collections/{collection_id}/orders` with an additional `GET /orders` that lists all orders with limited metadata. Not sure yet, but posting it here for discussion.

Reply:
Resolving this in favor of https://github.com/Open-EO/openeo-api/pull/447#discussion_r924812905